packages/create-astro/src/actions/dependencies.ts

```diff
@@ -39,6 +44,45 @@ export async function dependencies(
 			},
 			while: () => install({ packageManager: ctx.packageManager, cwd: ctx.cwd }),
 		});
+
+		let add = ctx.add;
+		if (!add && !ctx.yes) {
+			let { adapter } = await ctx.prompt({
+				name: 'adapter',
+				type: 'select',
+				label: title('adapter'),
+				message: 'Add an adapter',
```

Comment:
```suggestion
				message: 'Add an official adapter',
```
Don't really feel strongly about this, but more noting that since there *are* community adapters, we *might* get PRs or issues about "can I get my adapter added here?" that we can avoid by either something like adding "official" as shown above 

OR, after "none", a message pointing to the integrations catalogue for community adapters like,. "No adapter added. You can also find community adapters to install at https://astro.build/integrations!" or something like that. (The way we used to say, "No problem! about Typescript)

Reply:
That's fine with me, since there's not many community adapter I think we can consider adding the message separately from this PR